buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -14,13 +14,12 @@ object Dependencies {
     }
 
     object Accompanist {
-        private const val version = "0.19.0"
+        private const val version = "0.20.2"
```

Comment:
If you use compose beta04, please use accompanist 0.21.4-beta:
https://github.com/google/accompanist/releases/tag/v0.21.4